lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -236,8 +236,14 @@ Task cancelClaim(String taskId)
   Task forceCancelClaim(String taskId)
       throws TaskNotFoundException,
           InvalidOwnerException,
-          NotAuthorizedOnWorkbasketException,
-          InvalidTaskStateException;
+          InvalidTaskStateException {
+      try {
+        return this.cancelClaim(taskId, true);
+      } catch (NotAuthorizedOnWorkbasketException e) {
+        throw new SystemException(
+        "this should not have happened. You've discovered a new bug! :D", e); 
+      }
+    }
```

Comment:
You have to do this in `TaskServiceImpl#forceCancelClaim` ;) 

Please also remove the NotAuthorizedOnWorkbasketException from this interface method.

Reply:
Sorry for the confusion @mustaphazorgati , I returned the file I edited back to the original form and added my changes to the file you mentioned: TaskServiceImpl.java

Reply:
No worries mate! :) That's what PRs are for